data/secscan_model/secscan_v4_model.py

```diff
@@ -294,6 +294,14 @@ def mark_manifest_unsupported(manifest):
                 logger.exception("Failed to perform indexing, security scanner API error")
                 continue
 
+            if report["state"] == IndexReportState.Index_Finished:
+                index_status = IndexStatus.COMPLETED
+            elif report["state"] == IndexReportState.Index_Error:
+                index_status = IndexStatus.FAILED
+            else:
+                # Unknown state don't save anything
+                continue
```

Comment:
What are these states? Is the client expected to retry in these cases?

Reply:
If something goes wrong during the indexing the last state is returned, i.e. "scanLayers", these should be retried and when they are, Clair will pick up where it left off.